cmd/kots/cli/admin-console-push-images.go

```diff
@@ -67,8 +94,45 @@ func AdminPushImagesCmd() *cobra.Command {
 	cmd.Flags().String("registry-username", "", "user name to use to authenticate with the registry")
 	cmd.Flags().String("registry-password", "", "password to use to authenticate with the registry")
 
+	cmd.Flags().Bool("credentials-from-secret", false, "set this flag to use credentials storred in a secret by an existing kotsadm instance")
```

Comment:
why not just default to using this if the user didn't provide any creds? maybe even for the upgrade command?